docs/python/debugging.md

```diff
@@ -55,11 +55,214 @@ Selecting the **Node.js: Gulp task** yields the following result:
 
 See [Debugging specific app types](#debugging-specific-app-types) for details on all of these configurations.
 
-During debugging, the Status Bar shows the current configuration on the lower left; to its right is the current debugging interpreter. Selecting the configuration brings up a list from which you can choose a different configuration:
+During debugging, the Status Bar shows the current configuration and the current debugging interpreter. Selecting the configuration brings up a list from which you can choose a different configuration:
 
 ![Debugging Status Bar](images/debugging/debug-status-bar.png)
 
-By default, the debugger uses the same `python.pythonPath` workspace setting as for other features of VS Code. To use a different interpreter for debugging specifically, set the value for `pythonPath` in `launch.json` for the applicable debugger configuration as described in the next section. Alternately, select the named interpreter on the Status Bar to select a different one, which updates `python.pythonPath`.
+By default, the debugger uses the same `python.pythonPath` workspace setting as for other features of VS Code. To use a different interpreter for debugging specifically, set the value for `python` in `launch.json` for the applicable debugger configuration as described in the next section. Alternately, select the named interpreter on the Status Bar to select a different one, which updates `python.pythonPath`.
+
+## Basic debugging
+
+The simplest way to begin debugging a Python file is to use the **Run** view and click the **Run and Debug** button. When no configuration has been previously set, you will be presented with a list of debugging options. Select the appropriate option to quickly begin debugging your code.
+
+![Run the debugger](images/debugging/debug-run.png)
+
+Two common options are to use the **Python File** configuration to run the currently open Python file or to use the **Attach using Process ID** configuration to attach the debugger to a process that is already running.
+
+For information about creating and using debugging configurations see the [Initialize configurations](#initialize-configurations) and [Additional configurations](#additional-configurations) sections. Once a configuration is added, it can be selected from the dropdown list and started using the **Start Debugging** button.
+
+![Start debugging](images/debugging/debug-start-button.png)
+
+## Command line debugging
+
+The debugger can also be run from the command line. The debugger command line syntax is as follows:
+
+```bash
+python -m debugpy --listen [<address>:]<port>
+    [--wait-for-client]
+    [--configure-<name> <value>]...
+    [--log-to <path>] [--log-to-stderr]
+    <filename> | -m <module> | -c <code> | --pid <pid>
+    [<arg>]...
```

Comment:
It might be worth pointing out here that `<arg>...` is the command line arguments that will get passed to the app being launched as is.

Reply:
Will be updated in next commit.